### ISSUE TITLE
[Face Reading / FR205] Validate calibrated full-oval proxy on independent identities

### ISSUE THREAD

**Issue by gycha0109-beep** [gycha0109-beep/Saju#980]
FR204 completed a reproducible unseen-geometry intervention holdout and selected the calibrated full face-oval envelope as the next-stage candidate.

## Frozen candidate

- provider measurement: roll-normalized MediaPipe full official face-oval X envelope
- frozen multiplicative factor: `0.8185802384926992`
- candidate selection source: FR204 only
- no refit is permitted in FR205

FR204 exact-head reproducibility:
- dedicated run `35444952619`
- 20 deterministic intervention selections
- 17 independent references ready
- provider success 17/17
- raw mean absolute relative error: **20.75%**
- calibrated mean absolute relative error: **8.50%**
- calibrated median absolute relative error: **5.43%**
- Pearson exact vs calibrated: **0.8998**
- Spearman exact vs calibrated: **0.9167**

## FR205 question

Does that already-frozen candidate preserve facial-width magnitude/order on identities never used anywhere in FR199–FR204?

## Hard independence rule

Every FR205 identity must be absent from:
- FR199 public corpus;
- FR200/FR201 photo/OBJ experiments;
- FR202 exact same-mesh correspondence;
- FR203 calibration discovery;
- FR204 intervention holdout.

The candidate definition, factor, renderer policy, and evaluation metrics are frozen before observing FR205 oracle/reference results.

## Evidence hierarchy

Preferred:
1. paired image + physical/scan 3D ground-truth mesh for new identities.

Fallback research evidence only:
2. new public 2D identities evaluated independently by two 3D reconstruction oracles (MICA and 3DDFA_V2), with MediaPipe hidden from each oracle path.

Oracle consensus is secondary evidence and must not be described as scan ground truth.

## Dataset/access gate

Do not silently use a gated/non-commercial dataset as product training data.

Current known candidates require an explicit access/licence decision:
- Florence 2D/3D: paired 2D/3D but request-gated;
- MICA unified registrations: academic/non-commercial and constituent datasets separately gated;
- Bosphorus: paired textured 3D/2D candidate, access/licence must be verified before execution;
- AST-Face / FaceVerse public material is insufficient as a direct independent-identity cohort in current public form.

FR205 first executable step is therefore an **acquisition/runtime gate**, not another threshold-fitting experiment.

## Acceptance reporting

For any valid independent cohort, record without refitting:
- calibrated absolute relative error distribution;
- Pearson and Spearman ordering;
- fail-closed provider/reference failures;
- subgroup/source receipts where legally and technically available.

No numeric production threshold is authorized in advance by FR205.

## Authority

```text
providerIndexAdmissionAuthorized=false
anatomicalZygionClaimAuthorized=false
numericAcceptanceThresholdAuthorized=false
calibrationAuthorized=false
classifierAuthorized=false
traditionalProjectionAuthorized=false
productionAuthorized=false
commerceAuthorized=false
independentIdentityValidationComplete=false
```

FR205 may only change `independentIdentityValidationComplete` after a genuinely independent cohort has been executed and reviewed. It does not automatically authorize production/commerce.

Stack base: FR204 / PR #972.
No merge without explicit approval.

**Comment by gycha0109-beep** [comment]
FR205 independent-identity acquisition gate executed successfully without opening a PR, so no repository-wide PR CI fan-out was added.

Branch: `research/fr205-independent-identity-acquisition`
Head: `7a7df2d3cd83a3f209b490e736f628bb27cba6b6`
Run: `35445309740`
Job: `105903112659`
Artifact: `10584373666`
Digest: `sha256:153daf85e762e7fba9c099d9d696e7d004920ded27847adf9378c358783045cd`

Gate PASS:
- AST-Face pinned public demo exposes exactly 84 finite XYZ landmarks.
- Demo textured OBJ shape is executable: 42,774 vertices / 245,874 UV rows / 81,958 faces.
- MTL↔texture binding is intact.
- AST-Face repository contract confirms public standardized geometry/landmarks, while paired raw scans + synchronized RGB require controlled access via OSF account + signed DUA.
- MICA pinned license confirms non-commercial scientific-research-only use and separate FLAME access requirements.
- 3DDFA_V2 code is MIT, but its bundled modified BFM2009 asset is documented academic-only / commercial BFM license required.

Decision:
- preferred independent-identity validation path = AST-Face controlled-access raw scan + frontal RGB subset;
- frozen full-oval factor remains `0.8185802384926992`;
- no refit is allowed;
- independent identity validation cannot execute from currently public material alone;
- MICA / current 3DDFA+BMF stack remain research-oracle-only and are not product dependencies.

Authority remains false:
`independentIdentityValidationComplete=false`
`calibrationAuthorized=false`
`productionAuthorized=false`
`commerceAuthorized=false`.

This is now an external-access boundary, not an unresolved implementation problem.

**Comment by gycha0109-beep** [comment]
FR205A public independent-identity geometry holdout is now complete.

Final exact experiment:
- head `b639acb8728ca6be2be162647dd0305caebb6781`
- run `35447505605`
- job `105908873958`
- artifact `10585771111`
- artifact digest `sha256:9a6ee714cd27fc38c240b3cba489b6d7db787bfdeb5e2872e60a678fd44227ef`
- 98 selected / 98 independent face-breadth references ready / 98 MediaPipe successes / 0 provider failures

Public acquisition correction:
- AST-Face OSF public archive contains AU0 standardized mesh + 84-point XYZ landmarks for all 98 identities.
- synchronized real RGB remains unavailable publicly, so this is geometry-render evidence, not real-photo paired validation.

Reference:
- frame-aligned AST-Face 84-point full X-span;
- explicitly operational face breadth, not anthropometric zygion.

Final 98-identity result:
- raw full-oval mean ratio **0.9906**
- raw MAE **2.381%**
- raw median AE **2.210%**
- raw max AE **5.878%**
- Pearson **0.7763**
- Spearman **0.7267**

Frozen FR204 factor `0.8185802384926992` against this different face-breadth construct:
- mean ratio **0.8109**
- MAE **18.910%**
- median AE **18.729%**

Pilot-excluded `ast013`–`ast098` (86 untouched identities) reproduces the raw result:
- mean ratio **0.9906**
- MAE **2.388%**
- median AE **2.019%**
- max AE **5.878%**
- Pearson **0.7697**
- Spearman **0.7159**

Decision:
- FR204 factor remains construct-specific and is rejected as a generic face-breadth calibration.
- raw MediaPipe full face-oval width advances only as the next-stage operational face-breadth candidate.
- no anatomical zygion claim is made.
- real-photo validation remains open.

Authority remains fail-closed:
`independentRealPhotoValidationComplete=false`
`independentIdentityValidationComplete=false`
`anatomicalZygionClaimAuthorized=false`
`calibrationAuthorized=false`
`productionAuthorized=false`
`commerceAuthorized=false`.

Research note updated on branch head `dae0d11e983cbf21938370337860d56acc69c79c`. No merge performed.

**Comment by gycha0109-beep** [comment]
FR206 product-boundary correction supersedes FR205 as the active frontier.

Final interpretation of FR205 evidence:

- the 98-identity AST-Face result is retained as useful research evidence;
- raw full face-oval width is treated only as an **operational visible face-breadth candidate**;
- it is not anthropometric zygion or skeletal bizygomatic breadth;
- the FR204 factor `0.8185802384926992` is rejected as a generic face-breadth calibration;
- synchronized real-photo / scan-ground-truth zygion validation is **not a blocking prerequisite** for a separately named observable-morphology construct;
- no additional zygion correspondence campaign is the automatic next step.

Any later product validation of `visible_face_breadth` must test the declared operational construct (capture robustness, construct consistency, representative-image generalization) rather than re-open anatomical zygion proof unless the product explicitly makes an anatomical claim.

Active frontier: #1007 / FR206.

FR205 is complete as boundary-discovery research and is closed without merge.
